apps/ledger-live-mobile/src/components/RootNavigator/CustomBlockRouterNavigator.tsx

```diff
@@ -3,6 +3,23 @@ import { useEffect, useState } from "react";
 import { BehaviorSubject } from "rxjs";
 
 export const lockSubject = new BehaviorSubject<boolean>(false);
+export const exposedManagerNavLockCallback = new BehaviorSubject();
+
+export function useManagerNavLockCallback(): any {
```

Comment:
No idea how difficult this is but maybe we could declare the return type of this method